_opensearch/rest-api/snapshots/verify-snapshot-repository.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Verify Snaphot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 4
+---
+
+## Verify snapshot repository
+
+Verifies that a snapshot repository is functional. Verifies the repository on each node in a cluster.
+
+If verification is successful, the verify snapshot repository API returns a list of nodes connected to the snapshot repository. If verification failed, the API returns an error.
+
+If you use the security plugin, you must have the `manage cluster` privilege.
+{: .note}
+
+### Path parameters
+
+Path parameters are optional. 
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| repository | String | Name of repository to verify. |
+
+### Query parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| cluster_manager_timeout | Time | Amount of time to wait for a connection to the master node. Optional, defaults to `30s`. |
+| timeout | Time | The period of time to wait for a response. If a response is not received before the timeout value, the request fails and returns an error. Defaults to `30s`. |
+
+#### Sample request
+
+The following request verifies that the my-opensearch-repo is functional:
+
+````json
+POST /_snapshot/my-opensearch-repo/_verify?timeout=0s&cluster_manager_timeout=50s
+````
+
+### Response fields
+
+| Field | Data Type | Description | 
+:--- | :--- | :---
+| nodes | Object | A list (not an array) of nodes connected to the snapshot repository. Each node itself is a property where the node ID is the key and the name has an ID (Object) and a name (String). |
+
+#### Sample response
```

Comment:
Please move the Sample response section to right after Sample request. It makes sense to keep them together. Thanks!

Reply:
I have made this change.